Dockerfile

```diff
@@ -1,19 +1,25 @@
 # vim:set ft=dockerfile:
 FROM condaforge/mambaforge
-MAINTAINER https://github.com/bird-house/finch
+ARG DEBIAN_FRONTEND=noninteractive
+ENV PIP_ROOT_USER_ACTION=ignore
+LABEL org.opencontainers.image.authors="bourgault.pascal@ouranos.ca"
```

Comment:
Why not leave https://github.com/bird-house/finch as maintainer?  Not sure Pascal will want to receive all bugs report or requests.